packages/types/src/fulfillment/service.ts

```diff
@@ -337,6 +351,18 @@ export interface IFulfillmentModuleService extends IModuleService {
     sharedContext?: Context
   ): Promise<ShippingOptionTypeDTO>
 
+  /**
+   * Retrieve a fulfillment
+   * @param id
+   * @param config
+   * @param sharedContext
+   */
+  retrieveFulfillment(
```

Comment:
Should this be named `retrieve` only?

Reply:
no the main entity entry point is fulfillmentSet which differ from fulfillment